<?xml version="1.0"?>
<launch>
  <!-- Example launch file for the PhaseSpace ImpulseX2E motion-tracker
       calibration visualization.
       This assumes that you have already started the PhaseSpace OWL
       server and the ROS phasespace node(s). You might also need to
       check the PhaseSpace alignment transform and publish that.
       Also, set the colormap_scale to a distance that is typical
       for the calibration errors in your setup; e.g. value=0.01
       results in full-scale blue/red for distance errors of 0.01m
       or 1 centimeter.
  -->


  <node pkg="tams_phasespace" name="calibration_visualizer" type="calibration_visualizer" output="screen">
    <param name="debug_level" type="int" value="2" />
    <param name="parent_frame" type="string" value="world" />
    <param name="marker_radius" type="double" value="0.04" />
   
    <param name="update_rate" type="double" value="480.0" />
    <param name="visualization_rate" type="double" value="10" />
    <param name="colormap_scale" type="double" value="0.01" />

    <remap from="/calibration_visualizer/owl_markers" to="/phasespace/owl_markers" />
  </node>



</launch>
